Cannot find module './src/ClickToComponent' or its corresponding type declarations.

Hey everyone :) 
I encountered a small TypeScript problem when working with react-click-to-component:

**Describe the bug**
Running the typescript compiler with "skipLibCheck": false will result in the following error:
```
node_modules/click-to-react-component/src/types.d.ts:1:34 - error TS2307: Cannot find module './src/ClickToComponent' or its corresponding type declarations.
```

**To Reproduce**
Steps to reproduce the behavior:
1. Setup a new React project with TypeScript and use <ClickToComponent />
2. Make sure "skipLibCheck" is set to false
3. Run the TypeScript compiler

**Expected behavior**
No TSC errors should occur.

**Screenshots**
<img width="1475" alt="image" src="https://user-images.githubusercontent.com/12122390/166668532-77989160-3450-4ce9-b34f-52b42e92b2d9.png">


**Additional context**
I guess this happens, because the path reference inside of types.d.ts does not exist. Since this path should be resolved relative to the current file, it looks for a src dir inside of src and will not find that.

A possible fix would be to move types.d.ts to the root directory, or to fix the path.

--- a/.changeset/lovely-deers-mate.md
+++ b/.changeset/lovely-deers-mate.md
@@ -1,0 +1,6 @@
+---
+"next": patch
+"click-to-react-component": patch
+---
+
+Fix "Cannot find module './src/ClickToComponent' or its corresponding type declarations."

--- a/apps/next/tsconfig.json
+++ b/apps/next/tsconfig.json
@@ -3,7 +3,7 @@
     "target": "es5",
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
-    "skipLibCheck": true,
+    "skipLibCheck": false,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,

--- a/packages/click-to-react-component/src/types.d.ts
+++ b/packages/click-to-react-component/src/types.d.ts
@@ -1,4 +1,4 @@
-export { ClickToComponent } from './src/ClickToComponent'
+export { ClickToComponent } from './ClickToComponent'
 
 export type Editor = 'vscode' | 'vscode-insiders'
 

